src/content/add-to-app/ios/project-setup.md

```diff
@@ -5,75 +5,71 @@ description: Learn how to integrate a Flutter module into your existing iOS proj
 ---
 
 Flutter UI components can be incrementally added into your existing iOS
-application as embedded frameworks. There are a few ways to embed Flutter 
-in your existing application.
-
-1. **Use the CocoaPods dependency manager and installed Flutter SDK.**
-  In this case, the `flutter_module` is compiled from
-  the source each time the app is built. (Recommended.)
-
-1. **Create frameworks for the Flutter engine, your compiled Dart code,
-  and all Flutter plugins.** Here, you manually embed the frameworks,
-  and update your existing application's build settings in Xcode.
-  This can be useful for teams that don't want to require every developer
-  to have the Flutter SDK and Cocoapods installed locally.
-
-1. **Create frameworks for your compiled Dart code,
-  and all Flutter plugins. Use CocoaPods for the Flutter engine.** 
-  With this option, embed the frameworks for your application
-  and the plugins in Xcode, but distribute the
-  Flutter engine as a CocoaPods podspec.
-  This is similar to the second option, but it provides
-  an alternative to distributing the large Flutter.xcframework.
-
-For examples using an app built with UIKit, 
-see the iOS directories in the [add_to_app code samples][]. 
-For an example using SwiftUI, see the iOS directory in [News Feed App][].
+application as embedded frameworks.
+To embed Flutter in your existing application,
+consider one of the following three methods.
+
+**Use the CocoaPods dependency manager and installed Flutter SDK.**
+: Each time you build the app,
+  it compiles the `flutter_module` from source.
+  (Recommended.)
+
+**Create frameworks for each component.**
+: Create frameworks for the Flutter engine, your compiled Dart code,
+  and all Flutter plugins in Xcode.
+  Embed these frameworks and update your existing application's build settings.
+  This method works for teams that don't want to require every developer
+  to install the Flutter SDK and Cocoapods on their local machines.
```

Comment:
Could we move the drawbacks of option A to option A's paragraph? If you're skimming quickly, it looks like option A is recommended and has no drawbacks unless I happen to read this entire section.